tests/tests/datm_ocn_ice_mx100_gefs

```diff
@@ -0,0 +1,19 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_COLD test
```

Comment:
Once has to dig through the file to know what forcing this test uses.

All 4 files here have the same # CPLD_DATM_MOM6_CICE_CMEPS_COLD

Reply:
Done.